### PR TITLE
[SYCL][ESIMD][EMU] Setting default platform for ESIMD_EMULATOR

### PR DESCRIPTION
**Pull request by dongkyunahn-intel**
- CM_EMU library package is recently updated to support multiple
targets. This fix is to avoid target platform setting failure

### CODE REVIEW

File: SYCL/lit.cfg.py [hunk 1]
```diff
@@ -368,6 +368,8 @@
         if lit_config.params.get('ze_debug'):
             gpu_run_substitute = " env ZE_DEBUG={ZE_DEBUG} SYCL_DEVICE_FILTER=level_zero:gpu,host ".format(ZE_DEBUG=config.ze_debug)
             config.available_features.add('ze_debug'+config.ze_debug)
+    elif config.sycl_be == "ext_intel_esimd_emulator":
+        gpu_run_substitute += "CM_RT_PLATFORM=skl "
```
Comment (kbobrovs) [inline SYCL/lit.cfg.py]: please add comments what this setting means - `CM_RT_PLATFORM=skl`

Comment (dongkyunahn-intel) [reply SYCL/lit.cfg.py]: Done.

**Review by kbobrovs — APPROVED** [review]

**Comment by dongkyunahn-intel** [comment]
CM_EMU requires environment variable setup for `CM_RT_PLATFORM`. There was a bug in CM_EMU that allows kernel execution without the environment variable setup and it was fixed recently.

https://github.com/intel/cm-cpu-emulation/blob/master/README_CONFIG.md#hw-configuration-choice

**Comment by dongkyunahn-intel** [comment]
Failures are only on Windows - not related to this patch.

**Review by v-klochkov — APPROVED** [review]

**Pull request merged**